srcpkgs/uucp/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'uucp'
+pkgname=uucp
+version=1.07
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr --mandir=/usr/share/man --with-newconfigdir=/etc/uucp --sbindir=/usr/bin"
```

Comment:
all but newconfigdir are part of build-style

Reply:
The original folder is /usr/conf/uucp, the most logical path is /etc/uucp, as they are configuration files.

Reply:
I meant --prefix and others are passed due to build-style anyway, no need to define them here.

Reply:
Done.